spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkScan.java

```diff
@@ -130,6 +180,58 @@ public void testEstimatedRowCount() throws NoSuchTableException {
     assertThat(stats.numRows().getAsLong()).isEqualTo(10000L);
   }
 
+  @TestTemplate
+  public void testColStats() throws NoSuchTableException {
```

Comment:
Also needs tests for parameter on/vs off.

Reply:
added tests for `ENABLE_COLUMN_STATS` and `CBO` off